Bassic JavaScript/86-Using Objects for Lookups.js

```diff
@@ -0,0 +1,21 @@
+// Setup
+function phoneticLookup(val) {
+    var result = "";
+  
+    // Only change code below this line
+    var lookup= {
+      "alpha": "Adams",
+      "bravo": "Boston",
+      "charlie": "Chicago",
+      "delta":  "Denver",
+      "echo": "Easy",
+      "foxtrot": "Frank"
+    };
```

Comment:
var arr = [{id: 4, name: "Pedro"}, {id: 2}, ... , {id: N}]
var obj = { "Pedro" : { id: 4} , "Andres" : { id: 2}, ... , N : { id: N } }
var n = 50
function findNumber(n, arr) {
for(...)
  return arr[i].id === 300
}
function findNumberObj(n, obj) {
 return obj[n].id
}